delete/wp-site

```diff
@@ -9,8 +9,11 @@ fi
 
 cd "$(dirname $0)"
 # Read config file with paths to WP-installs and usernames
-source ../conf
+source /usr/local/default/etc/wp-mass-admin/wpma.conf
 
+if [[ -e /usr/local/etc/wp-mass-admin/wpma.conf ]]; then
+    source /usr/local/etc/wp-mass-admin/wpma.conf
+fi
```

Comment:
Should we not reach some failure state if there is no conf? Perhaps we should also check in `/etc/wp-mass-admin` before bailing?